cluster/addons/rbac/cluster-loadbalancing/glbc/roles.yaml

```diff
@@ -43,7 +43,7 @@ rules:
   verbs: ["update"]
 - apiGroups: ["extensions", "networking.k8s.io"]
   resources: ["ingresses/status"]
-  verbs: ["update"]
+  verbs: ["patch"]
```

Comment:
leave update in place as well. if you have one, you effectively have the other, and leaving both in place allows a smooth transition from a previous version which used update

Reply:
Good call!  Done